XPCBackupService/Entities/BackupTreeNode.swift

```diff
@@ -30,9 +30,9 @@ class BackupTreeNode {
     var backupTotalPogress: Progress
     var rootBackupFolder: URL
     var syncRetries: UInt64 = 0
-    var backupRealm: BackupRealmProtocol
+    var backupRealm: any BackupRealmProtocol
```

Comment:
is better to be specific here, this should use something like `BackupSyncNodeRepository` rather than any protocol